dialog-polyfill.js

```diff
@@ -493,15 +493,13 @@
     dialog.close(returnValue);
   }, true);
 
-  window['dialogPolyfill'] = dialogPolyfill;
-  dialogPolyfill['forceRegisterDialog'] = dialogPolyfill.forceRegisterDialog;
```

Comment:
These are for the Closure compiler, so it doesn't inline them. They should probably stay up here, regardless of how the code is used.